plugins/restapi/v2/plugin.go

```diff
@@ -106,6 +107,10 @@ const (
 	// RouteControlSnapshotsCreate is the control route to manually create a snapshot files.
 	// POST creates a snapshot (full, delta or both).
 	RouteControlSnapshotsCreate = "/control/snapshots/create"
+
+	// RouteComputeWhiteFlag is the debug route to compute the white flag confirmation for the cone of the given parents.
+	// POST computes the white flag confirmation.
```

Comment:
```suggestion
	// RouteComputeWhiteFlagMutations is the route to compute the white flag mutations for the cone of the given parents.
	// POST computes the white flag mutations.
```

Can you also move the whole variable up, between `RouteReceiptsMigratedAtIndex` and `RoutePeer`?
So we have the same order as in iota.go